packages/file_selector/file_selector_web/pubspec.yaml

```diff
@@ -22,8 +22,8 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  mockito: ^4.1.1
-  pedantic: ^1.8.0
+  mockito: ^5.0.0-nullsafety.5
+  pedantic: ^1.10.0-nullsafety.3
```

Comment:
I think this shouldn't have been pushed in this change. Apologies, I just noticed!

Reply:
Oof, not sure how I missed that. Want to push a quick revert on of this?

Reply:
Here's the revert: https://github.com/flutter/plugins/pull/3528

Reply:
I'm sorry my bad 🥺

Reply:
@SirusCodes not at all, I should have published my comment wayyyyy earlier :)